containers/docker-in-docker/.devcontainer/library-scripts/README.md

```diff
@@ -0,0 +1,5 @@
+# Warning: Folder contents may be replaced
+
```

Comment:
@Chuxel  Is there anything I need to do additionally in setup for the comment in this readme to be true?

Reply:
Nope - as long as the name in the `/script-library` folder matches the `.devcontainer/library-scripts` folder, it will be automatically updated if the `/script-library` file is updated after a CI run.